docs/en/latest/references/apisix_consumer_v2.md

```diff
@@ -0,0 +1,46 @@
+---
+title: ApisixConsumer/v2
+keywords:
+  - APISIX ingress
+  - Apache APISIX
+  - ApisixConsumer
+description: Reference for ApisixConsumer/v2 custom Kubernetes resource.
+---
+<!--
+#
+# Licensed to the Apache Software Foundation (ASF) under one or more
+# contributor license agreements.  See the NOTICE file distributed with
+# this work for additional information regarding copyright ownership.
+# The ASF licenses this file to You under the Apache License, Version 2.0
+# (the "License"); you may not use this file except in compliance with
+# the License.  You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+-->
+
+## Spec
+
+See the [definition](../../../../samples/deploy/crd/v1/ApisixConsumer.yaml) on GitHub.
+
+| Field            | Type    | Description                                                                                                                                    |
+|------------------|---------|------------------------------------------------------------------------------------------------------------------------------------------------|
+| authParameter          | object   | Configuration of one of the available authentication plugins.                                                                                                   |
+| authParameter.basicAuth.value   | object   | Plugin configuration for [basic-auth plugin](https://apisix.apache.org/docs/apisix/plugins/basic-auth/)                                      |
```

Comment:
In this doc, could you please wrap all these auth plugin names in description (e.g. `basic-auth`, `key-auth`) in inline code?

Reply:
```suggestion
| authParameter.basicAuth.value   | object   | Plugin configuration for [`basic-auth` plugin](https://apisix.apache.org/docs/apisix/plugins/basic-auth/)                                      |
```
like this